scenarios/model-catalog/samples/github/azure-openai/gpt-4o/js/azure_ai_inference/getting_started.md

```diff
@@ -0,0 +1,246 @@
+
+
+# Getting Started
+
+- [Create a personal access token](#create-a-personal-access-token)
+- [Install dependencies](#install-depedencies)
+- [Set environment variables](#set-environment-variables)
+- [Run a basic code sample](#run-a-basic-code-sample)
+- [Explore more code snippets](#explore-more-samples)
+
+## 1. Create a personal access token
+
+You'll need to **[create a token](https://github.com/settings/tokens)** to enable free API access. The token will be sent to a Microsoft service but does not need any permissions.
+
+## 2. Install dependencies
+
+Install Azure AI Inferencing package using the following command:
+
+```
+WORK IN PROGRESS
+```
+
+## 3. Set environment variables
+Update or create an environment variable to set your token as the key for the client code.
+
+```bash
+export TOKEN="<your-key-goes-here>"
+
+```
+Get the model endpoint url and use it in the code below by exporting it as an environment variable
+
+```bash
+export MODEL_ENDPOINT="<your-model-endpoint-goes-here>"
+```
+
+Set model name in an env variable:
+
+```bash
+export MODEL_NAME=gpt-4o
+```
+
+## 4. Run a basic code sample
+
+This sample demonstrates a basic call to the chat completion API.
+It is leveraging your endpoint and key. The call is synchronous.
+
+
+```js
+import ModelClient from "@azure-rest/ai-inference";
+import { AzureKeyCredential } from "@azure/core-auth";
+
+const endpoint = process.env["MODEL_ENDPOINT"];
+const token = process.env["TOKEN"];
+const modelName = process.env["MODEL_NAME"];
+
+export async function main() {
+
+  const client = new ModelClient(endpoint, new AzureKeyCredential(token));
+
+  const response = await client.path("/chat/completions").post({
+    headers: { "x-ms-model-mesh-model-name": modelName }, // NOTE: this is a temporary hotfix
+    body: {
+      messages: [
+        { role:"system", content: "You are a helpful assistant." },
+        { role:"user", content: "What is the capital of France?" }
+      ],
+      model: modelName
+    }
+  });
+
+  if (response.status !== "200") {
+    throw response.body.error;
+  }
+  console.log(response.body.choices[0].message.content);
+}
+
+main().catch((err) => {
+  console.error("The sample encountered an error:", err);
+});
+```
+
+
+## 5. Explore more samples
+
+For a better user experience, you will want to stream the response
+of the model so that the first token shows up early and you avoid waiting for long responses.
+
```

Comment:
Not clear to me why we talk about streaming here (or only streaming), and then again we have a section below about streaming.